### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by shraddha-amber**
This PR contains:

Sign in and Signup pages
Private route

### CODE REVIEW

File: src/features/Auth/authSlice.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        "smLoginData",
+        JSON.stringify({
+          token: action.payload.encodedToken,
+          user: action.payload.foundUser,
+        })
+      );
+    },
+    [signInUser.rejected]: (state, action) => {
+      state.isLoading = false;
+      console.log(action.payload);
```
Comment (znjs) [inline src/features/Auth/authSlice.js]: can remove the console

File: src/features/Auth/authSlice.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        "smLoginData",
+        JSON.stringify({
+          token: action.payload.encodedToken,
+          user: action.payload.createdUser,
+        })
+      );
+    },
+    [signUpUser.rejected]: (state, action) => {
+      state.isLoading = false;
+      console.log(action.payload);
```
Comment (znjs) [inline src/features/Auth/authSlice.js]: can remove the console

**Review by znjs — APPROVED** [review]
- Good PR
- Clean code
- Functionality working as expected

**Review by muskaanshah — APPROVED** [review]
Clean code, I like how you've made constants for the routes.

File: src/features/Auth/authSlice.js (hunk 1 above)
Comment (shraddha-amber) [reply src/features/Auth/authSlice.js]: thanks for pointing out @znjs

**Pull request merged**